src/Microsoft.Health.Dicom.Functions/Indexing/QueryTagIndexingOptions.cs

```diff
@@ -6,14 +6,15 @@
 using System;
 using System.ComponentModel.DataAnnotations;
 
-namespace Microsoft.Health.Dicom.Core.Configs
+namespace Microsoft.Health.Dicom.Functions.Indexing
 {
     /// <summary>
     /// Represents the configuration for a "reindex" function.
     /// </summary>
-    public class ReindexOperationConfiguration
+    public class QueryTagIndexingOptions
     {
-        public const string SectionName = "Reindex";
+        internal const string ConfigurationSectionName = "Indexing";
```

Comment:
Any reason for renaming? Will it include setting for both reindexing and deindexing?

Reply:
I am trying to differentiate between things that are "configurations" and things that are "options." I can probably leave this name alone, but throughout this change I'm trying to draw this line between the data model (options) and how they're recorded (config).